guide/how-it-works/ecash/fedimint.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: guide
+title: Fedimint
+description: Learn the basics of Fedimint.
+nav_order: 3
+parent: Ecash
+grand_parent: How it works
+permalink: /guide/how-it-works/ecash/fedimint/
+redirect_from:
+ - /guide/ecash/fedimint
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/fedimint.jpg
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Explains generally how fedimint works
+
+Illustration sources
+
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/fedimint.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/fedimint@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/fedimint-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/fedimint-mobile@2x.jpg"
+   alt-text = "ecash overview illustration"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Fedimint
+Fedimint is the first bitcoin-backed ecash protocol. Fedimint decentralizes trust across a federation of guardians, ensuring that no single guardian has complete control over user funds. Fedimint allows for various Federation configurations, each catering to a community's specific needs and trust relationships
+
+## Federated mints
```

Comment:
there are three diagrams directly after each other, each discussing a different topic. It could be helpful to have short explainers for each of them, so that readers don't need to rely entirely on their own interpretation. It could also help to visually separate and structure the content.